draft-ietf-masque-h3-datagram.md

```diff
@@ -76,46 +78,48 @@ document are to be interpreted as described in BCP 14 {{!RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
 
-# Flow Identifiers {#flow-id}
+# Flows
 
-Flow identifiers represent bidirectional flows of datagrams within a single QUIC
-connection. These are conceptually similar to streams in the sense that they
-allow multiplexing of application data.  Flows lack any of the ordering
-or reliability guarantees of streams.
+Flows are bidirectional exchanges of datagrams within a single QUIC connection.
+These are conceptually similar to streams in the sense that they allow
+multiplexing of application data. Flows are identified within a connection by a
+numeric value, referred to as the flow ID.  A flow ID is a 62-bit integer (0
```

Comment:
```suggestion
numeric value, referred to as the flow identifier (or flow ID).  A flow ID is a 62-bit integer (0
```

Reply:
This text is liberally copied from QUIC transport stream definition. Im happy to diverge but just wanted to point that out.

Reply:
I can live with flow ID for consistency